build/checkstyle/tachyon_checks.xml

```diff
@@ -217,6 +217,14 @@
     </module>
     <module name="MissingDeprecated"/>
     <module name="EqualsHashCode"/>
+    <!--
```

Comment:
would it make sense to introduce this only once it does not need to be commented out?